dashboard/src/get_elements.py

```diff
@@ -138,29 +164,50 @@ def raw_elements_to_elements(self, api_response):
             elem['classifications'] = list(map(lambda class_id: raw_elements['classifications'][class_id], elem['classifications']))
             elem['property_sets'] = list(map(lambda pset_id: raw_elements['property_sets'][pset_id], elem['psets']))
             del elem['psets']
-            self.elements[elem['uuid']] = elem
+            self.elements.append(elem)
 
     def filter_by_types(self):
         if self.ifc_type:
             self.elements = [elem for elem in self.elements.values() if elem['type'] == self.ifc_type]
             return
         self.elements = [elem for elem in self.elements.values()]
 
+    def define_column_types(self):
+        for key in self.types_ref:
+            c = Counter(self.types_ref[key])
+            self.df = self.df.astype({key: c.most_common()[0][0]})
+
+    def fill_void(self):

```

Comment:
Why do we need to fill void? 0.0 is very different than no value

Reply:
It was in the type casting process, when it was float it needed to have 0.0 assigned